### PR TITLE
Troubleshooting and Subroutine Support sections added

### PR DESCRIPTION
**Pull request by zeibura**
Troubleshooting and Subroutine Support sections added to Readme file for the next release.

### CODE REVIEW

**Review by grianbrcom — APPROVED** [review]
Looks good.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ Autocomplete speeds up the coding process by intuitively suggesting the most lik
 - Code Snippets
 - Copybook variable and paragraph names
 - Names of copybooks that are used in the program
+- Names of locally-stored subroutines
```
Comment (temanbrcom) [inline README.md:48]: Only names of subroutines that are used in the program at the moment

**Review by asatklichov — APPROVED** [review]

**Review by temanbrcom — APPROVED** [review]
🍰

**Review by zimlu02 — APPROVED** [review]
nice!

**Comment by temanbrcom** [comment]
Please, squash it on merge

**Pull request merged**